test/gyppies-test.js

```diff
@@ -7,6 +7,8 @@ const fs = require('fs');
 const path = require('path');
 const rimraf = require('rimraf');
 const ninja = require('ninja.js');
+const debug = require('debug')('gyp.js');
+debug.log = console.log.bind(console);
```

Comment:
Literally did this in the last 20 minutes. I'm trying to get [Appyover to CI this Windows style](https://ci.appveyor.com/project/refack/get-vs2017-path)
I'll clean in up a bit.